src/content/quickstart/vsphere_aws/_index.md

```diff
@@ -0,0 +1,170 @@
+---
+date: 2020-02-21T13:36:18+01:00
+title: "OpenShift with Service Discovery (vSphere/AWS)"
+weight: 15
+---
+
+In this quickstart guide, we shall cover the necessary steps to deploy OpenShift Container Platform (OCP) on vSphere and AWS. Once the OCP clusters are deployed, we shall cover how to deploy Submariner and connect the two clusters.
+
+### OpenShift Prerequisites
+
+Before we proceed, the following prerequisites have to be downloaded and added to your PATH
+
+ 1. [openshift-installer](https://cloud.redhat.com/openshift/install/aws/installer-provisioned) 
+ 2. [pull secret](https://cloud.redhat.com/openshift/install/aws/installer-provisioned)
+ 3. [oc](https://cloud.redhat.com/openshift/install/aws/installer-provisioned) tools
+ 4. [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
+
+{{% notice info %}}
+
+Please ensure that the tools you downloaded above are compatible with the OpenShift Container Platform version.
+
+{{% /notice %}}
+
+### Deploy Cluster on vSphere (OnPrem)
+
+Create the necessary infrastructure on vSphere and ensure that your machines have direct internet access before starting the installation.
+To deploy OCP 4.4, follow the instructions shown [here](https://docs.openshift.com/container-platform/4.4/installing/installing_vsphere/installing-vsphere.html)
+
+Assuming that you deployed the cluster (say, cluster-a) with default network configuration, the Pod and Service CIDRs would be
+
+| Pod CIDR     | Service CIDR |
+|--------------|--------------|
+|10.128.0.0/14 |172.30.0.0/16 |
+
+Submariner creates a VxLAN overlay network in the local cluster and uses port 4800/UDP to encapsulate traffic from the worker nodes to the gateway nodes to preserve the source-ip for the inter-cluster traffic.
+Ensure that firewall configuration on vSphere cluster allows 4800/UDP across all the worker nodes. 
+
+|  Protocol  |  Port  |     Description                              |
+|------------|--------|----------------------------------------------|
+|   UDP      |  4800  | overlay network for inter-cluster traffic    |
+
+{{% notice tip %}}
+
+Although we are using default OCP network configuration on vSphere, you can install vSphere with custom network customization as shown [here](https://red.ht/2WFjEVg)
+
+{{% /notice %}}
+
+### Deploy Cluster on AWS
+
+#### Configure AWS CLI with appropriate values
+
+```bash
+$ aws configure
+AWS Access Key ID [None]: ....
+AWS Secret Access Key [None]: ....
+Default region name [None]: ....
+Default output format [None]: text
+```
+
+For more details, follow this link:
+
+> https://docs.openshift.com/container-platform/4.4/installing/installing_aws/installing-aws-account.html
+
+In this step we shall deploy cluster-b on AWS, modifying the default Cluster and Service CIDRs
```

Comment:
```suggestion
In this step we shall deploy cluster-b on AWS and modify the default Cluster and Service CIDRs
```

Reply:
Slightly modified to reflect that we will modify first and then deploy the cluster.
Note: We cannot modify the configuration after deployment.